pages/css/components/dropdown.md

```diff
@@ -186,3 +186,29 @@ Align the direction of dropdown menus and their arrows with modifier classes.
   </div>
 </details>
 ```
+
+### Dark
+
+```html
+<div class="bg-gray-dark p-4 mt-n4 ml-n4 mr-n4" style="min-height: 400px;">
+  <details class="dropdown details-reset details-overlay d-inline-block">
+    <summary class="btn" aria-haspopup="true">
+      Dropdown
+      <div class="dropdown-caret"></div>
+    </summary>
+
+    <div class="dropdown-menu dropdown-menu-se dropdown-menu-dark">
+      <div class="dropdown-header">
+        Dropdown header
+      </div>
+      <ul>
+        <li><a class="dropdown-item" href="#url">Dropdown item</a></li>
+        <li><a class="dropdown-item" href="#url">Dropdown item</a></li>
+        <li class="dropdown-divider" role="separator"></li>
```

Comment:
Can this be a `span` instead of an `<li>`? `role=separator` helps with semantics and a11y but I think this would be best if it weren't a list item so that the screen reader doesn't count it in its list of items.

Reply:
Or an `<hr>` element might fit? Probably needs extra styling.